### PR TITLE
QOL-9055 Prepare for Amazon Linux 2

### PR DESCRIPTION
**Pull request by ThrawnCA**
- Update CKAN plugins to support Python 3
- Update cookbook to handle Amazon Linux 2 structure

### CODE REVIEW

File: .docker/test-OpenData.ini [hunk 1]
```diff
@@ -97,7 +97,7 @@ ckan.redis.url = redis://redis:6379
 #       Add ``datapusher`` to enable DataPusher
 #       Add ``resource_proxy`` to enable resource proxying and get around the
 #       same origin policy
-ckan.plugins = stats text_view image_view recline_view datastore data_qld scheming_datasets validation dcat qa archiver report data_qld_reporting data_qld_google_analytics harvester_data_qld_geoscience harvest qgovext ytp_comments datarequests xloader csrf_filter resource_type_validation
+ckan.plugins = stats text_view image_view recline_view datastore data_qld data_qld_google_analytics scheming_datasets validation dcat qa archiver report harvester_data_qld_geoscience harvest qgovext ytp_comments datarequests xloader csrf_filter resource_type_validation ssm_config odi_certificates
```
Comment (duttonw) [inline .docker/test-OpenData.ini:100]: so data_qld_reporting is not needed anymore?

File: test/features/data_qld_theme.feature [hunk 2]
```diff
@@ -85,11 +85,6 @@ Feature: Theme customisations
         Then I should see an element with xpath "//a[contains(@href, '/datastore/dump/') and contains(@href, 'format=json') and contains(string(), 'JSON')]"
         Then I should see an element with xpath "//a[contains(@href, '/datastore/dump/') and contains(@href, 'format=xml') and contains(string(), 'XML')]"
 
-    @Publications
-    Scenario: Homepage 'Publish in the Gazettes' link is correct
```
Comment (duttonw) [inline test/features/data_qld_theme.feature:89]: why purge? should it not be moved to the publication_qld_theme.feature instead?

**Review by duttonw — APPROVED** [review]

File: test/features/data_qld_theme.feature (hunk 2 above)
Comment (ThrawnCA) [reply test/features/data_qld_theme.feature:89]: It's already in `homepage.feature`

File: .docker/test-OpenData.ini (hunk 1 above)
Comment (ThrawnCA) [reply .docker/test-OpenData.ini:100]: It's just a placeholder these days (literally, the entry in `setup.py` is `data_qld_reporting=ckanext.data_qld.plugin:PlaceholderPlugin`). `data_qld` covers it.

**Pull request merged**